src/client/src/ui/spotTile/model/spotTileModel.js

```diff
@@ -213,4 +224,13 @@ export default class SpotTileModel extends ModelBase {
         }
     }
   }
+
+  _startChartIQTimer(){
+    this.chartIQisOpening = true;
+    setTimeout(this._stopChartIQTimer.bind(this), 2000);
```

Comment:
should probably use the schedule service for this as it wraps it up in a disposable we can manage locally